app/assets/javascripts/admin/all.js

```diff
@@ -60,7 +60,7 @@
 //= require moment/nb.js
 //= require moment/pt-br.js
 //= require moment/sv.js
-//= require ../shared/mm-foundation-tpls-0.8.0.min.js
+//= require ../shared/mm-foundation-tpls-0.8.1.min.js
```

Comment:
Looks good! I think we must copy/move this fork to the OFN org somewhere like http://github.com/openfoodfoundation/angular-foundation

Reply:
Good idea to fork it to our organisation. There also seems to be another fork that had a couple of useful commits: https://github.com/cwadrupldijjit/angular-foundation

We could also open our pull request against that repository and see if it's maintained. If not, it could be useful to cherry-pick those commits for our fork.

I'm not sure if @kristinalim has organisation permissions to create new repositories though. That may be something we have to do. But are you happy to open another pull request with the other repository first and see if that guy would like to be the new maintainer?

Reply:
I can reassign the repository to OFN, no problem. I wouldn't need prior permission, but OFN would need to approve the transfer of ownership within one day.

I actually submitted a pull request to the repository already: https://github.com/yalabot/angular-foundation/pull/319 (Oops! There's a merge conflict with `0.9.0-SNAPSHOT` there - I will fix that.) But there is [hint](https://github.com/yalabot/angular-foundation/issues/311) that the repository will no longer be maintained.

Reply:
Ah, I didn't realize that you meant I could try opening a pull request at https://github.com/cwadrupldijjit/angular-foundation instead. :slightly_smiling_face: I will do that. Thanks, @mkllnk!

Reply:
Here are the PRs to the original repository and another fork:

* https://github.com/yalabot/angular-foundation/pull/319
* https://github.com/cwadrupldijjit/angular-foundation/pull/1

I created a release [`0.9.0-20180826174721`](https://github.com/kristinalim/angular-foundation/releases/tag/0.9.0-20180826174721) which is based on the "master" branch in https://github.com/cwadrupldijjit/angular-foundation/ and cherry-picks the modal fixes. (The compiled JS is in the `gh-pages` branch, following the process for the `angular-foundation` project.) This is the version which this PR now uses.

The instructions about transferring repositories which I read must have been for non-organization Github accounts. I couldn't initiate a transfer to `openfoodfoundation`. Would a fork of my fork will be sufficient, while we're waiting to see if any of the PRs could get merged?